### PR TITLE
SCMO-9746 Create cookie response plugin

### PR DESCRIPTION
**Pull request by aswirski-cloudentity**

### CODE REVIEW

**Comment by aswirski-cloudentity** [comment]
Please help me out making sure this plugin is activated in AcceptanceTest ... seems like it's inactive, probably must be registered in some place.

**Comment by jdabrowski** [comment]
We would also need documentation for this plugin similar to what we have for others

**Comment by aswirski-cloudentity** [comment]
ok, I will do it now

**Comment by jdabrowski** [comment]
acceptance test fixed

**Review by jdabrowski — APPROVED** [review]

File: docs/plugins/transform-response-cookie.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          "method": "GET",
+          "pathPattern": "/transform-some-cookies",
+          "responsePlugins": [
+            {
+              "name": "transform-response-cookie",
+              "conf": {
+                "name": "originalCookieName",
+                "domain": "optional.domain.com",
+                "path": "/optional/path/",
+                "body": {
```
Comment (mnikliborc) [inline docs/plugins/transform-response-cookie.md:34]: "body" should be not part of 'conf', compare with TransformResponseCookieConf case class

**Review by mnikliborc — APPROVED** [review]

**Pull request merged**